cities/helsinki/assets/js/piwik.js

```diff
@@ -0,0 +1,11 @@
+var _paq = _paq || [];
+_paq.push(["setDomains", ["*.kerrokantasi.hel.fi"]]);
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+(function() {
+  var u="//analytics.hel.ninja/piwik/";
```

Comment:
This should be changed to //analytics.hel.ninja/
and below should be piwik.php -> matomo.
Neither are blockers for this PR though, compatibility aliases are on place for the foreseeable future.

Reply:
The entire Piwik thing should be made optional as well, I did however not have time to do that right now. I will ask someone to look into it for a future PR.

Regarding the address, I have not changed the content of that file and it was like this before these changes, so this is not really an issue with this PR as far as I'm concerned 🙂